site/data/dynamo.ts

```diff
@@ -287,7 +288,12 @@ export const getOrganisationsInfo = async (): Promise<Organisations | null> => {
     return parsedOrg.data;
 };
 
-export const deletePublishedDisruption = async (disruption: FullDisruption, disruptionId: string, id: string) => {
+export const deletePublishedDisruption = async (
+    disruption: FullDisruption,
+    disruptionId: string,
+    id: string,
+    isTemplate?: boolean,
```

Comment:
There's a few more places we have to check isTemplate to pass the correct table name:
Line 323 (we'll also need to conditionally change the SK again based on whether isTemplate is true)
Lines 339, 352, 361

Reply:
done